src/ontology/mondo-edit.obo

```diff
@@ -615764,6 +615766,15 @@ is_a: MONDO:0006552 {source="https://orcid.org/0000-0002-0819-0473"} ! folliculi
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0819-0473
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/pull/8094" xsd:anyURI
 
+[Term]
+id: MONDO:1060107
+name: CYP7B1-related disorder of oxysterol accumulation
+def: "Any disorder of oxysterol accumulation caused by biallelic loss of function variants in the CYP7B1 gene." [https://clinicalgenome.org/affiliation/40097/]
```

Comment:
Not to be nick-picking (but I will be): this definition is ok, but we don't have a term for "disorder of oxysterol accumulation" in Mondo. Therefore, we need to define what it is. 
Option number 1: "Any lipid metabolism disorder characterized by (_add definition of oxysterol accumulaiton disorder_) and caused by biallelic...."

Option number 2: "Any disorder of oxysterol accumulation caused by (...). A disorder of oxysterol accumulation is (_definition of oxysterol accumulaiton disorder_)